client/src/pages/Search.jsx

```diff
@@ -49,39 +49,39 @@ export default function Search() {
           width: "clamp(300px,60%,600px)",
           height: "clamp(300px,60%,600px)",
         }}
+        onSubmit={(e) => {
+          e.preventDefault();
+          handleApi();
+        }}
       >
-        <FormControl sx={{ width: "100%" }} className="d-outline">
+        <FormControl sx={{ width: "100%" }} className="">
           {loading && (
             <div>
               <i class="fa-solid fa-seedling fa-bounce"></i>Loading...
             </div>
           )}
           {!loading && (
-            <>
+            <Stack direction="column" gap={2}>
               <TextField
                 className="searchTextField"
                 placeholder="Search Plants"
                 onChange={(e) => setSearch(e.target.value)}
                 value={search}
+                required
+                min="1"
               />
-              <Button variant="contained" color="primary" onClick={handleApi}>
+              <Button variant="contained" color="primary" type="submit">
                 Click to search
               </Button>
-            </>
+            </Stack>
           )}
         </FormControl>
 
         {/* show results */}
 
         <Stack direction={"column"} gap={3} py={3}>
           {data &&
-            data.map((result) => (
-              <SearchResultDetail
-                imgLink={result?.item?.Img}
-                title={result?.item?.["Latin name"]}
-                data={result}
```

Comment:
this is data={result?.item} in dev branch. I need this stuff to stay how it is in dev.